src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -263,10 +246,35 @@ public Task<ExecutionResult> TryExecuteCommand()
 "#load                " + InteractiveWindowResources.LoadHelp
         };
 
+        internal List<string> GenerateShortcutDescriptions()
+        {
+            var list = new List<string>();
+            list.Add("Enter                " + InteractiveWindowResources.EnterHelp);
+            list.Add("Ctrl-Enter           " + InteractiveWindowResources.CtrlEnterHelp1);
+            list.Add("                     " + InteractiveWindowResources.CtrlEnterHelp2);
+            list.Add("Shift-Enter          " + InteractiveWindowResources.ShiftEnterHelp);
+            list.Add("Escape               " + InteractiveWindowResources.EscapeHelp);
+            list.Add("Alt-UpArrow          " + InteractiveWindowResources.AltUpArrowHelp);
+            list.Add("Alt-DownArrow        " + InteractiveWindowResources.AltDownArrowHelp);
+            list.Add("Ctrl-Alt-UpArrow     " + InteractiveWindowResources.CtrlAltUpArrowHelp);
+            list.Add("Ctrl-Alt-DownArrow   " + InteractiveWindowResources.CtrlAltDownArrowHelp);
+            if (UseSmartUpDown)
+            {
+                list.Add("UpArrow              " + InteractiveWindowResources.UpArrowHelp1);
+                list.Add("                     " + InteractiveWindowResources.UpArrowHelp2);
+                list.Add("DownArrow            " + InteractiveWindowResources.DownArrowHelp1);
+                list.Add("                     " + InteractiveWindowResources.DownArrowHelp2);
+            }
```

Comment:
Hmm, I'm not sure. To me, the interactive window is just like any other VS text window, so I simply expect up and down arrows to behave like that unless help text says otherwise.